dbt/adapters/rockset/impl.py

```diff
@@ -22,10 +22,10 @@
 OK = 200
 NOT_FOUND = 404
 ASYNC_OPTIONS = {
-                "client_timeout_ms": 1000,  # arbitrary
-                "timeout_ms": 1800000,
-                "max_initial_results": 1,
-            }
+    "client_timeout_ms": 1000,  # arbitrary
+    "timeout_ms": 1800000,
+    "max_initial_results": 1,
+}
```

Comment:
We may as well convert this to the new async API at this point: https://rockset.com/docs/rest-api/#query